wordpress-mobile/WordPress-Android
# Feature/6313 app banner handling

Fixes #6313

To test:

Go to the following sections of WordPress.com with the app installed and logged in

1. Reader and verify when clicking the app banner it goes to the Reader
2. Stats and verify when clicking the app banner it goes to stats
3. Editor and verify when clicking the app banner it goes to the editor
4. Notifications and verify when clicking the app banner it goes to notifications
